Photon Collection Updates and debugging

skyriacoCMS opened the pull request:
I have included some more photon variables, cleaned the existing ones and fixed a bug. 

-Savvas

skyriacoCMS committed:
Photon Collection Updates and debugging

dmajumder commented:
@skyriacoCMS Just wondering, did you check the size of the events after these changes?

skyriacoCMS commented:
Hello Devdatta,

I added 6 variables and removed an old one. Thus total of 5 new floats per
photon, I do not expect any special size increase.

Cheers.

_Savvas Kyriacou_
*Department of Physics and Astronomy         *
*Rutgers, The State University of New Jersey *

*CERN B8 R-020       ,__,           (o.o)           /),,,,)            "  "
      *

On Tue, Oct 11, 2016 at 10:54 AM, dmajumder notifications@github.com
wrote:

> @skyriacoCMS https://github.com/skyriacoCMS Just wondering, did you
> check the size of the events after these changes?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> https://github.com/cmsb2g/B2GAnaFW/pull/62#issuecomment-252851749, or mute
> the thread
> https://github.com/notifications/unsubscribe-auth/AHCcD2edfJ6NrAjumWOn4zsnaVlcP-A_ks5qy07bgaJpZM4KTBTe
> .
